services/common/src/components/forms/RenderFileUpload.tsx

```diff
@@ -1,47 +1,69 @@
-import React from "react";
+/* eslint-disable */
```

Comment:
I always have to ask why when I see that eslint is disabled for the whole file!
Was it a specific rule? Is it something we think we can clean up in the future that's just too much scope for now?

Reply:
Definitely not intended, will remove. Copied it over from `FileUpload.js` and didn't notice 🙃 
![image](https://github.com/bcgov/mds/assets/66635118/c3719577-1dde-4554-aed4-1626e1873c21)